style.css

```diff
@@ -99,3 +102,216 @@ p {
   display: flex;
   justify-content: center;
 }
+
+.works_section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
- [ ] The main container of the projects is supposed to have `display:grid`. Kindly fix that